kalite/main/api_views.py

```diff
@@ -10,9 +10,33 @@
 from fle_utils.internet.webcache import backend_cache_page
 
 from kalite.topic_tools import get_topic_tree
+from kalite.topic_tools.content_recommendation import get_resume_recommendations, get_next_recommendations, get_explore_recommendations
+from kalite.facility.models import FacilityUser
 
 @api_handle_error_with_json
 @backend_cache_page
 def topic_tree(request, channel):
     parent = request.GET.get("parent")
     return JsonResponse(get_topic_tree(channel=channel, language=request.language, parent=parent))
+
+@api_handle_error_with_json
+def content_recommender(request):
```

Comment:
So we moved away from the Tastypie endpoints into plain views, correct? Because tastypie is such a complex beast, and we're moving away from it soon?